scikitplot/decomposition.py

```diff
@@ -156,14 +166,36 @@ def plot_pca_2d_projection(clf, X, y, title='PCA 2-D Projection', ax=None,
     classes = np.unique(np.array(y))
 
     colors = plt.cm.get_cmap(cmap)(np.linspace(0, 1, len(classes)))
-
+    
+    vectors=np.transpose(clf.components_[0:2, :])
+    
+    xs = transformed_X[:,0]
+    ys = transformed_X[:,1]   
+    
+    
     for label, color in zip(classes, colors):
         ax.scatter(transformed_X[y == label, 0], transformed_X[y == label, 1],
                    alpha=0.8, lw=2, label=label, color=color)
+        
+    if biplot:  
+        for i in range(vectors.shape[0]):
+            ax.annotate("", xy=(vectors[i,0]*max(xs), 
+                        vectors[i,1]*max(ys)), xycoords='data',
+                        xytext=(0, 0), textcoords='data', 
+                        arrowprops={'arrowstyle': '-|>', 'ec': 'r'})
+
+            if feature_labels is None:
+                plt.text(vectors[i,0]*max(xs)*1.05, vectors[i,1]*max(ys)*1.05,
+                         "Variable"+str(i), color='b', fontsize=text_fontsize)
+            else:
+                plt.text(vectors[i,0]*max(xs)*1.05, vectors[i,1]*max(ys)*1.05,
+                        feature_labels[i], color='b', fontsize=text_fontsize)
```

Comment:
Style

Reply:
I think the if-else branch here should only set a single variable `feature_label` since the rest of the calculations are duplicated.

Reply:
Actually this is cleaner

```
plt.text(vectors[i,0]*max(xs)*1.05, vectors[i,1]*max(ys)*1.05,
feature_labels[i] if feature_labels else "Variable" + str(i), color='b', fontsize=text_fontsize)
```